strategies/outputs_file.py

```diff
@@ -1,15 +1,26 @@
-exec(open('header.py').read())
+from configparser import SafeConfigParser
 import os
-
+import sys
 
 def outpath_websource(websourcechoice):
+    parser = SafeConfigParser()
+    parser.read('./setup/user.ini')
+    candidate='OS'
+    print(parser.has_section(candidate))
+    user_OS=parser.get('OS', 'user_OS')
+    user_OS=user_OS[1:-1]
+    print (user_OS)
 
     if websourcechoice=='journal_en_francais_facile' or 'AZ_lyrics':
-        outpath_loc=output_dir1
         if user_OS=='Windows':
-            outpath = outpath_loc+'\\'+ websourcechoice +'\\'
-        if user_OS=='Ubuntu':
-            outpath = outpath_loc+'/'+websourcechoice +'/'
+            spacer_char='\\'
```

Comment:
you can delete this code and use 

https://docs.python.org/3.2/library/os.html#os.sep